ci/prow/periodic_config.go

```diff
@@ -327,3 +328,71 @@ func generateClearAlertsPeriodicJob() {
 	addVolumeToJob(&data.Base, "/secrets/cloudsql/monitoringdb", "monitoring-db-credentials", true, "")
 	executeJobTemplate("periodic clearalert", readTemplate(periodicCustomJob), "presubmits", "", data.PeriodicJobName, false, data)
 }
+
+func generateIssueTrackerPeriodicJobs() {
+	staleJobName := "periodic-test-infra-stale"
```

Comment:
The "test-infra" in the job names make me think that these job only act on knative/test-infra, which is not true. Please use the same naming convention as the other generic jobs, like "ci-knative-issue-tracker-stale".

Reply:
Done